.github/workflows/compute.yml

```diff
@@ -28,22 +28,23 @@ jobs:
     environment: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')) && 'main' || 'development' }}
 
     steps:
-      - uses: actions/checkout@v4
-
-      - name: setup node
-        uses: actions/setup-node@v4
+      - uses: actions/checkout@v5
         with:
-          node-version: "20.10.0"
+          token: ${{ inputs.authToken }}
+          repository: ${{ fromJson(inputs.eventPayload).repository.full_name }}
+          fetch-depth: 0
+          persist-credentials: false
 
-      - name: execute directive
-        run: node ./dist/index.js
-        id: command-ask
+      - name: Run Claude Code
+        uses: ubiquity-whilefoo/claude-code-action@custom-event
         env:
-          KERNEL_PUBLIC_KEY: ${{ secrets.KERNEL_PUBLIC_KEY }}
-          PLUGIN_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-          SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
-          VOYAGEAI_API_KEY: ${{ secrets.VOYAGEAI_API_KEY }}
-          OPENROUTER_API_KEY: ${{ secrets.OPENROUTER_API_KEY }}
-          UBIQUITY_OS_APP_NAME: ${{ secrets.UBIQUITY_OS_APP_NAME }}
-          GOOGLE_SERVICE_ACCOUNT_KEY: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_KEY }}
+          CUSTOM_EVENT_PAYLOAD: ${{ inputs.eventPayload }}
+          CUSTOM_EVENT_NAME: ${{ inputs.eventName }}
+        with:
+          mode: "tag"
+          trigger_phrase: "(@UbiquityOS|@ubiquityos|/ask)"
+          allowed_tools: "Bash(gh issue view:*),Bash(gh pr view:*)"
```

Comment:
Perhaps we can allow everything basically? It would be very interesting to ask it to do more things.